src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,202 @@
+        [InlineData("##other", "ns1 ns2", "ns3", 1)]
+        //[Variation(Desc = "complextype Any ns - ns1 ns3, attrgroup Any ns1 ns2, not allow ns3 attribute")]
+        [InlineData("ns1 ns3", "ns1 ns2", "ns3", 1)]
+        //[Variation(Desc = "complextype Any ns - ns1 ns3, attrgroup Any ns1 ns2, not allow ns2 attribute")]
+        [InlineData("ns1 ns3", "ns1 ns2", "ns2", 1)]
+        //[Variation(Desc = "complextype Any ns - ns1 ns3, attrgroup Any ns1 ns2, allow ns1 attribute")]
+        [InlineData("ns1 ns3", "ns1 ns2", "ns1", 0)]
+        //[Variation(Desc = "complextype Any ns - ##other, attrgroup Any ##other, not allow ns1 attribute")]
+        [InlineData("##other", "##other", "ns1", 1)]
+        public void v1(string ns1, string ns2, string attrNs, int expectedError)
+        {
+            Initialize();
+            XmlSchemaSet xss = new XmlSchemaSet();
+            xss.XmlResolver = new XmlUrlResolver();
+            xss.ValidationEventHandler += new ValidationEventHandler(ValidationCallback);
+            xss.Add(GetIntersectionSchema(ns1, ns2, attrNs));
+            xss.Compile();
+
+            Assert.Equal(errorCount, expectedError);
+        }
+
+        [Theory]
+        //[Variation(Desc = "basetype Any ns - ##any, derivedType Any ns - ns1, allow ns2 attribute")]
+        [InlineData("##any", "ns1", "ns2", 0)]
+        //[Variation(Desc = "basetype Any ns - ns1, derivedType Any ns - ##any, allow ns2 attribute")]
+        [InlineData("ns1", "##any", "ns2", 0)]
+        //[Variation(Desc = "basetype Any ns - ns1 ns2, derivedType Any ns - ns2 ns3 , allow ns3 attribute")]
+        [InlineData("ns1 ns2", "ns2 ns3", "ns3", 0)]
+        //[Variation(Desc = "basetype Any ns - ##other, derivedType Any ns - ##other , not allow current ns")]
+        [InlineData("##other", "##other", "ns1", 1)]
+        //[Variation(Desc = "basetype Any ns - ns1 ns2, derivedType Any ns - ##other , allow ns1")]
+        [InlineData("ns1 ns2", "##other", "ns1", 0)]
+        //[Variation(Desc = "basetype Any ns - ns1 ns2, derivedType Any ns - ##other , allow ns2")]
+        [InlineData("ns1 ns2", "##other", "ns2", 0)]
+        //[Variation(Desc = "basetype Any ns - ##other, derivedType Any ns - ns1 ns2 , allow ns2")]
+        [InlineData("##other", "ns1 ns2", "ns1", 0)]
+        //[Variation(Desc = "basetype Any ns - ##other, derivedType Any ns - ns1 ns2 , allow ns2")]
+        [InlineData("##other", "ns1 ns2", "ns2", 0)]
+        public void v2(string ns1, string ns2, string attrNs, int expectedError)
+        {
+            Initialize();
+            XmlSchemaSet xss = new XmlSchemaSet();
+            xss.XmlResolver = new XmlUrlResolver();
+            xss.ValidationEventHandler += new ValidationEventHandler(ValidationCallback);
+            xss.Add(GetUnionSchema(ns1, ns2, attrNs));
+            xss.Compile();
+
+            Assert.Equal(errorCount, expectedError);
```

Comment:
Should be `Assert.Equal(expectedError, errorCount)`, as it's `Assert.Equal(expected, actual)`